Wrappers/Python/examples/viewer_slider.py

```diff
@@ -0,0 +1,174 @@
+# inspired by https://examples.vtk.org/site/Python/Visualization/FroggieView/
+import vtk
+from ccpi.viewer import viewer2D
+
+
+class SliderProperties:
```

Comment:
Once you add the class in the widgets folder, most of the code here can be deleted as redundant. Please call the classes from "widgets" and simplify this example

Reply:
I don't think this is necessary and I would remove this file completely on merge